docs/content/guides/dagster/software-defined-assets.mdx

```diff
@@ -206,9 +206,9 @@ class LocalFileSystemIOManager(ConfigurableIOManager):
         if isinstance(obj, PandasDF):
             directory = self._get_fs_path(context.asset_key)
             os.makedirs(directory, exist_ok=True)
-            open(os.path.join(directory, "_SUCCESS"), "wb").close()
             csv_path = os.path.join(directory, "part-00000.csv")
             obj.to_csv(csv_path)
+            open(os.path.join(directory, "_SUCCESS"), "wb").close()
```

Comment:
do you know why this file is created in the first place? and why it isn't created when the `obj` is a `SparkDF`?

Reply:
I don't - I have zero context on this guide, as it predates me.

Reply:
looks like @sryza was the original author, so maybe he knows? my uneducated guess is that writing a `SparkDF` creates this file automatically, so we create it ourselves for pandas DFs for consistency

Reply:
ok! is there a functional reason to keep it around? or is it just consistency in the file system? If we can write a short explanatory comment i think that could be helpful